packages/serve-cli/Dockerfile

```diff
@@ -0,0 +1,30 @@
+# IMPORTANT: make sure bundle is ready with `yarn bundle`
+
+# we build with node 21 because node-libcurl has published prebuilt releases (building from source fails)
```

Comment:
Node.js 21 active support ended three months ago, security support ended one month ago

Reply:
Please read the full dockerfile, Node 21 is just for the build phase - later we use Node 22.

Reply:
Makes sense. 😄 Let's keep track of it and see what we can do to push support for Node.js 22 in node-libcurl?

Reply:
Building with GCC is failing when building from source. I tried figuring it out, but ended spending too much time so I just did this 😅 .

It could be that some internals did change for Node 22 (or maybe the image). Officially libcurl doesnt support Node 22, so it could be that too.

Reply:
What are we using libcurl for by the way?

Reply:
It's used in whatwg-node and it allows for HTTP/2 support out-of-box and I guess performance too (better mem too I reckon).